__tests__/index.test.js

```diff
@@ -180,3 +180,46 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("status 200: responds with the updated article object", () => {
+      return request(app)
+          .patch("/api/articles/1")
+          .send({inc_votes: 1})
+          .expect(200)
+          .then(({body}) => {
+              expect(body.votes).toBe(101);
+          });
+  });
+
+  //Error handling
+  test("status 404: responds with error message when the given article_id does not exist", () => {
+      return request(app)
+          .patch("/api/articles/1000")
+          .send({inc_votes: 1})
+          .expect(404)
+          .then(({body}) => {
+              expect(body.msg).toBe("Article not found");
+          });
+  });
+
+  test("status 400: responds with error message when given an invalid article_id", () => {
+      return request(app)
+          .patch("/api/articles/invalid")
+          .send({inc_votes: 1})
+          .expect(400)
+          .then(({body}) => {
+              expect(body.msg).toBe("Bad request");
+          });
+  });
+
+  test("status 400: responds with error message when given an invalid inc_votes", () => {
+      return request(app)
+          .patch("/api/articles/1")
+          .send({inc_votes: "invalid"})
+          .expect(400)
+          .then(({body}) => {
+              expect(body.msg).toBe("Bad request");
+          });
+  });
+});
```

Comment:
I'd also add a test here with the wrong key for inc_votes and an empty body.

Reply:
Great implementation of the feedback.